frontend/src/components/cfdtables/CfdTableMaker.tsx

```diff
@@ -1,236 +0,0 @@
-import { CheckIcon, ChevronRightIcon, ChevronUpIcon, CloseIcon } from "@chakra-ui/icons";
```

Comment:
The reason why I introduced a separate file (`CfdTableMaker`) for the creating a table for the maker was because the table to accept/reject orders should contain different information than the other tables.

Now (with this PR) the maker sees values like `Unrealized P/L` when he does not care about it as it will always be 0 at this time. Also, he sees `margin` which is the taker's margin but does not know about his own values (because we don't have these values in our model yet).

What I want to say is: imho these tables cannot be merged.

Reply:
I am not sure I can follow your arguments, let's have a closer look at the values needed so the maker can take a decision if he should accept / reject (because that's what we need).

> Unrealized P/L

Yes, this one is unnecessarily in the accept/reject table now. But, it was actually part of the maker table and if it simplifies things I would not care to much about that.

> sees margin which is the taker's margin

No, each role sees their margin correctly. The margin currently always represents the own margin - we can also send the counterparty margin to the UI if needed. Should the maker also see the one of the taker? Is that relevant for decision making? (I think not...)

Things that were additionally displayed in the maker's table were:

* OrderId
* Leverage
* Timestamp

Does that maker need that for decision making? 
Given that we are currently always at a fixed leverage, I think we can introduce this table at a later point. 
Are the timestamp and the order id necessary for decision making?

Reply:
The `Unrealized P/L` was hidden (same for orderId, leverage and timestamp). Look further down below on line 190.

> No, each role sees their margin correctly. [..]

I wasn't aware of it. The maker definitely cares about both margins. Especially in a case where the maker wants to open a hedged position. Imho these values are crucial: 

- opening price
- taker's leverage and margins and liquidation price: to be able to make an informed decision if this position is too risky and to be able to open a hedged position
- maker's margin: to know how much the maker has to put up. 

-----

I'm fine with removing this if you think it simplifies things. Imho it's easier to have two different tables which can evolve separately.

Reply:
Removing this duplication simplifies code a lot and we did say initially that the maker's UI is not as important so I am happy to accept that certain values are there unnecessarily. If it just comes to filtering data, we can also easily build that on top of a single table component.

Reply:
I understand @bonomat's agruments, and I think we will need the second table eventually. I would still merge it with one for now, and eventually just duplicate that one and adapt as needed, once it is used. Since this table is fully functional it should be easy to copy and adapt as needed.